package.json

```diff
@@ -28,6 +28,10 @@
     {
       "name": "Stijn de Witt",
       "email": "StijnDeWitt@hotmail.com"
+    },
+    {
```

Comment:
Thanks a ton, i am not very aware of CSS Grids, so its best that you take care of this subcomponent hence forth